crates/fluvio-protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.11.0"
+version = "0.11.1"
```

Comment:
API seems to changed, this requires version to be updated to `0.12.0`

Reply:
The only thing that has changed in the fluvio-protocol is a unit test. Or do the producer's changes also affect the protocol?